sycl/doc/extensions/proposed/sycl_ext_oneapi_profiling_tag.asciidoc

```diff
@@ -151,13 +151,16 @@ The event's `info::event_profiling::command_submit` timestamp reflects the
 time at which `submit_profiling_tag` is called.
 The event's `info::event_profiling::command_end` timestamp reflects the time
 at which the event enters the "complete" state.
+The event's `info::event_profiling::command_start` timestamp reflects the time
+that the profiling tag command starts executing. This timestamp is between
+the `info::event_profiling::command_submit` and
+`info::event_profiling::command_end` timestamps.
```

Comment:
Please add a line break at the end of the sentences (here and in the paragraph below).  I'm experimenting with a new style, where we add a line break after each sentence, and the rest of this extension already follows that style.